docs/website/root/mithril/threat-model.md

```diff
@@ -0,0 +1,318 @@
+---
+sidebar_position: 3
+sidebar_label: Mithril Threat Model
+---
+
+# Mithril Threat Model
+
+:::danger
+
+This document is a draft version of the **Mithril threat model** prepared by the **Mithril core team**:
+- We expect to receive **external feedback and contributions** before we can consider it **final**.
+- Feel free to **contribute** to this document by using the **Edit this page** link a the bottom of the page.
+- A **GitHub discussion** is also available [here](https://github.com/input-output-hk/mithril/discussions/).
+
+::: 
+
+This document provides an analysis of the various security threats and possible mitigations that could affect the Mithril network and its participants. 
+
+This is an adversarial mindset document and aims to analyse the system from an attacker's perspective.
+
+The threat model is a living document and is kept up-to-date with the [latest Mithril version](https://github.com/input-output-hk/mithril/releases/latest)
+
+## System analysis
+
+### System description 
+
+:::info To do
+
+- Maybe reduce and move details into [architecture page](https://mithril.network/doc/mithril/mithril-network/architecture)?
+- Also we should update the [protocol page](https://mithril.network/doc/mithril/mithril-protocol/protocol)
+
+:::
+
+System consists of three main components: signers, aggregator and clients.
+
+Mithril signers do certify Cardano chain data using a Mithril signing key. That means, Mithril signers need access to a trusted `cardano-node` and the Mithril signing key to operate.
+
+Mithril signing keys are rotated every epoch and need to be certified by the Cardano KES key. For this, the Mithril signers need access to the KES key in order to register the signing key for this epoch. 
+
+Cardano KES keys are also used by block producing `cardano-node` and are typically located on the block producing machine.
+
+Cardano KES keys need to be [evolved every 36 hours](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/stake-pool-operations/7_KES_period.md), while they can be rotated from a root key when needed.
+
+:::info To do
+
+Is there a Cardano threat model about this?
+
+:::
+
+All Mithril signers and Mithril clients connect to a single aggregator using HTTP over TLS.
+
+Registering a Mithril signing key means that a signer sends its corresponding verification key to the aggregator, for the purpose of distribution to all other Mithril signers.
+
+A Mithril aggregator coordinates creation of signatures by all registered signers. Mithril signers do ask the aggregater whether a signature is pending on a regular basis. The aggregator responds with information what to sign and a list of public information about all registered signers.
```

Comment:
typo: ... Mithril signers do ask the **aggregator** whether...

Reply:
Done